tasks/compose.yml

```diff
@@ -2,7 +2,7 @@
   pip:
     name: docker-compose
     state: "{{ docker_compose_state }}"
-    version: "{{ docker_compose_version }}"
+    version: "{{ docker_compose_version if 'ubuntu2204' not in group_names else omit }}"
```

Comment:
The name is not the most correct, as other distributions will share the same problem and it's more attached to the version of docker. And please document this in the role README.